src/ArmTemplates/Common/API/Clients/Groups/Responses/GetAllGroupsLinkedToProductResponse.cs

```diff
@@ -0,0 +1,20 @@
+﻿// --------------------------------------------------------------------------
+//  <copyright file="GetAllGroupsLinkedToProductResponse.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation. All rights reserved.
+//  </copyright>
+// --------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Groups;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.API.Clients.Groups.Responses
+{
+    public class GetAllGroupsLinkedToProductResponse
+    {
+        [JsonProperty("value")]
```

Comment:
Wasn't this "Value" before?

Reply:
answered [here](https://github.com/Azure/azure-api-management-devops-resource-kit/pull/639#discussion_r823682497)